Documentation/security/policy/language.rst

```diff
@@ -503,6 +504,23 @@ but not CIDR prefix ``10.96.0.0/12``
 
         .. literalinclude:: ../../../examples/policies/l3/cidr/cidr.json
 
+.. _cidr_select_nodes:
+
+Selecting nodes with CIDR / ipBlock
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. include:: ../../beta.rst
+
+By default, CIDR-based selectors do not match in-cluster entities (pods or nodes).
+Optionally, the policy engine can be directed to select nodes by CIDR / ipBlock.
+This requires Cilium to be configured with ``--policy-cidr-match-mode=nodes`` or
+the equivalent Helm value ``policyCIDRMatchMode: nodes``. It is safe to toggle this
+option on a running cluster, and toggling the option affects neither upgrades nor downgrades.
+
+When this is enabled, every agent will allocate a distinct local
+:ref:`security identity <security_identities>` for all other nodes. This will cause
+a minor increase in memory usage on very large clusters.
```

Comment:
how large are we talking? Is it worth highlighting? Maybe take 5K nodes as a target, are we talking ~1MiB, 10MiB, 100MiB?

Reply:
@nathanjsweet did the actual benchmarking; what was the difference you saw? I can't recall.

(This line was added for box-checking.)

Reply:
Updated, thanks!